app/src/components/account/EmailAuthentication.vue

```diff
@@ -40,6 +44,9 @@ export default {
     goToNotifications() {
       this.$router.push({ name: "notifications" }).catch((err) => {})
     },
+    goToSignInModal() {
+      this.$router.push({ name: "sign-in-modal" }).catch((err) => {})
```

Comment:
This is not needed and in this form would hide errors...

Reply:
what do you mean by "hide errors"?